### ISSUE TITLE
a11y: Events in calendar cannot get keyboard focus

### ISSUE THREAD

**Issue by roseleaf** [zendesk/volunteer_portal#10]
Because the calendar event links are contained in spans, they are unreachable via keyboard.

This in contrast to the dates in the calendar, which are within <a> tags.
